tests/vertical_playground.html

```diff
@@ -666,7 +667,43 @@
           </value>
         </block>
       </category>
-      <category name="Data" colour="#FF8C1A"></category>
+      <category name="Data" colour="#FF8C1A">
```

Comment:
For now we can have this category for testing, but this will eventually be dynamically populated, through Blockly.Variables.flyoutCategory